src/articles/SearchField.js

```diff
@@ -5,25 +5,32 @@ import { ClearSearchButton } from "../components/allButtons.sc";
 
 export default function SearchField({ api, query }) {
   const [searchTerm, setSearchTerm] = useState(query);
+  const [isFocused, setIsFocused] = useState(false);
+
+  const handleFocus = () => setIsFocused(true);
+  const handleBlur = () => setIsFocused(false);
 
   function keyDownInSearch(e) {
     if (e.key === "Enter") {
       console.log(searchTerm);
       api.logUserActivity(api.SEARCH_QUERY, "", searchTerm, "");
-      window.location = `/articles?search=${searchTerm}`;
+      window.location = `/articles/search?search=${searchTerm}`;
```

Comment:
We now have a redirect function -> `utils/routing/routing.js`
Guess it would be more elegant to use that and be consistent.